volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -26,43 +26,113 @@
 
 
 class module(generic.GenericIntelProcess):
+    @property
+    def mod_mem_type(self):
+        """Return the mod_mem_type enum choices if available or None if not"""
+        if not self.has_member("_mod_mem_type"):
+            try:
+                vmlinux = linux.LinuxUtilities.get_module_from_volobj_type(
```

Comment:
This seems a very long winded way of achieving the goal?  I'd have expected `context.get_enumeration(self.get_symbol_table_name + constants.BANG + "mod_mem_type")` to work, or even `context.get_enumeration("mod_mem_type")`?

Reply:
Ah thanks - I'll give it a go!